packages/server-wallet/src/__chain-test__/directly-funded-channel.test.ts

```diff
@@ -127,26 +127,32 @@ it('Create a directly funded channel between two wallets ', async () => {
   });
 
   const postFundA = await postFundAPromise;
-  await channelFundedBPromise;
-  expect(getChannelResultFor(channelId, [postFundA.channelResult])).toMatchObject({
+  const postFundB = await postFundBPromise;
+
+  expect(postFundA.channelResult).toMatchObject({
     status: 'running',
     turnNum: 2,
   });
```

Comment:
This removes the channelId check that exists with `getChannelResultFor(channelId, [postFundA.channelResult])`
```suggestion
  expect(postFundA.channelResult).toMatchObject({
    channelId,
    status: 'running',
    turnNum: 2,
  });
```